### PR TITLE
[#1140] Add Multi Quick Fixes documentation to the "Eclipse Support".

### PR DESCRIPTION
**Pull request by miklossy**
Signed-off-by: Tamas Miklossy <miklossy@itemis.de>

### CODE REVIEW

**Comment by nbhusare** [comment]
LGTM

File: xtext-website/documentation/310_eclipse_support.md [hunk 1]
```diff
@@ -208,6 +208,21 @@ public void fixName(final Issue issue,
 }
 ```
 
+### Multi-Quickfixes
+Xtext also provides a Quickfix API for the definition of multi-quickfixes. It's implementation is quite similar to the quick fix above, but you need to call `acceptor.acceptMulti()` instead of `acceptor.accept()`.
```
Comment (cdietrich) [inline xtext-website/documentation/310_eclipse_support.md:212]: Its

File: xtext-website/documentation/310_eclipse_support.md [hunk 2]
```diff
@@ -208,6 +208,21 @@ public void fixName(final Issue issue,
...
+```java
+@Fix(IssueCodes.INVALID_FEATURE_NAME)
+public void fixFeatureName(Issue issue,
```
Comment (cdietrich) [inline xtext-website/documentation/310_eclipse_support.md:215]: would be nice to have an example for the second api too

Comment (cdietrich) [reply xtext-website/documentation/310_eclipse_support.md:215]: https://www.typefox.io/blog/xtext-2-13-0-released-semantic-editing-made-easy

File: xtext-website/documentation/310_eclipse_support.md (hunk 1 above)
Comment (miklossy) [reply xtext-website/documentation/310_eclipse_support.md]: Fixed.

File: xtext-website/documentation/310_eclipse_support.md (hunk 2 above)
Comment (miklossy) [reply xtext-website/documentation/310_eclipse_support.md:215]: To keep the documentation as small as possible, I think it is enough to have only one example that makes the user aware of that feature. Once they start using that Multi-Quickfix API, they will search for further documentation/examples and will find the blog post above.

**Pull request merged**